Dockerfile.QA

```diff
@@ -337,6 +337,10 @@ RUN pip3 install --upgrade wheel setuptools && \
 # Hence, fixing the boofuzz version to 0.3.0
 RUN pip3 install 'boofuzz==0.3.0'
 
+# For PA doc testing
```

Comment:
Looks like we can avoid adding it to the container and just put it in the test:

https://github.com/triton-inference-server/server/blob/main/qa/L0_doc_links/test.sh#L42